server/routes/api/integrations/integrations.ts

```diff
@@ -129,9 +141,32 @@ router.post(
     const { user } = ctx.state.auth;
     const { transaction } = ctx.state;
 
-    const integration = await Integration.findByPk(id, { transaction });
+    const integration = await Integration.findByPk(id, {
+      rejectOnEmpty: true,
+      transaction,
+    });
     authorize(user, "delete", integration);
 
+    // If it's GitHub, trigger its API for deletion of this installation
+    if (integration.service === IntegrationService.GitHub) {
+      const installationId = (integration as Integration<IntegrationType.Embed>)
+        .settings?.github?.installation.id;
+
+      if (env.GITHUB_APP_ID && env.GITHUB_APP_PRIVATE_KEY && installationId) {
```

Comment:
Yes, uninstall requires app auth as opposed to user auth 🙈

Reply:
Makes sense